### DISCUSSION TITLE
Use WHERE BINARY Clause with Prisma Client

### DISCUSSION THREAD

**Discussion by thopaw** [prisma/prisma#17917] [Q&A] [answered]
Hello,

I am struggeling with the `conatins` filter in my query on a MySql Database as it does not handle German Umlaute correctly.
Now I can change all encodings and collate to a option which suits my requirments better or I could query the data with

```
SELECT * 
WHERE BINARY text like '%search%'
````

Is there an option to use a WHERE BINARY query with the prisma client?

**Comment by nurul3101** [comment, marked as answer]
Hello @thopaw 👋 

For now, you can use [Raw Query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) to execute `WHERE BINARY` query. 

You should be able to use `WHERE BINARY` with PrismaClient once `whereRaw` feature request is implemented.
- https://github.com/prisma/prisma/issues/5560

**Reply by thopaw** [reply]
Ok thanks you for your answer. I will look into the raw query. Probably I will loose the type safety when I use raw queries, right?

**Reply by nurul3101** [reply]
You can type the raw query results as well: https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#typing-queryraw-results

**Comment by nurul3101** [comment]
Marking this as resolved.
